src/styles.css

```diff
@@ -0,0 +1,51 @@
+@import url("https://delta-ui-preview-2.netlify.app/library/component.css");
+
+/* @import "siteComponents/sidebar.css";
+@import "siteComponents/footer.css";
+@import "siteComponents/utils.css"; */
```

Comment:
remove the comment code.

Reply:
will be removed in upcoming refactoring.